src/chanjo2/meta/handle_d4.py

```diff
@@ -48,31 +48,38 @@ def intervals_coverage(
     return intervals_cov
 
 
-def evaluate_region_completeness(
+def evaluate_intervals_completeness(
     d4_file: D4File,
-    chromosome: str,
-    start: int,
-    stop: int,
+    intervals: List[Tuple[str, int, int]],
     completeness_threholds: Optional[List[int]],
 ) -> List[Tuple[int, Decimal]]:
-    """Use NumPy to Calculate the coverage completeness over a list of threshold values for a chromosomal region."""
+    """Use NumPy to calculate the coverage completeness over a list of threshold values for a chromosomal region."""
 
     if completeness_threholds is None:
         return []
 
-    region_size: int = stop - start
-    per_base_depth: ndarray = d4_file.load_to_np(f"{chromosome}:{start}-{stop}")
-
     completeness_values: List[Tuple[int, Decimal]] = []
+    total_region_length = sum([interval[2] - interval[1] for interval in intervals])
```

Comment:
```suggestion
    total_region_length: int = sum([interval[2] - interval[1] for interval in intervals])
```
I wonder if one could do something more readable like:
```
 total_region_length: int = sum([interval[CHR_STOP] - interval[CHR_START] for interval in intervals])
```